src/components/WorkOrder/RateScheduleItems/AddedRateScheduleItems.js

```diff
@@ -132,6 +138,11 @@ const AddedRateScheduleItems = ({
       <a className="lbh-link" href="#" onClick={addRateScheduleItem}>
         + Add another SOR code
       </a>
+      <br />
+      <br />
```

Comment:
Do we just want the two links on separate lines here? If so it's best to avoid using `<br />` as that is a specific use case for new lines amongst text. Could just put both links within their own `<div>`